docs/install_notes.rst

```diff
@@ -49,10 +49,10 @@ See the `virtual environment tutorial <https://docs.python.org/3/tutorial/venv.h
 	You can then run :command:`poetry shell` to create a terminal inside the
 	environment and run Scenic as usual.
 
-Scene Schematics Don't Appear
-+++++++++++++++++++++++++++++
+Scene Schematics Don't Appear (2D)
+++++++++++++++++++++++++++++++++++
 
-If no window appears when you ask Scenic to generate and display a scene (as in the example commands in :doc:`quickstart`), this means that Matplotlib has no `interactive backend <https://matplotlib.org/stable/users/explain/backends.html>`_ installed.
+If no window appears when you ask Scenic to generate and display a scene using the ``--2d`` flag (as in the example commands in :doc:`quickstart`), this means that Matplotlib has no `interactive backend <https://matplotlib.org/stable/users/explain/backends.html>`_ installed.
```

Comment:
Should use a cross-reference to the command-line arguments page, like `:option:--2d`